BlueprintUI/Sources/Element/UIViewElement.swift

```diff
@@ -123,7 +123,7 @@ public extension UIViewElement {
             }
             
             config.apply { view in
-                self.updateUIView(view, with: .init(isMeasuring: false))
+                self.updateUIView(view, with: .init(isMeasuring: false, environment: context.environment))
```

Comment:
Didn't push it yet, but another clever way to make useView have a shorter call site is to use #file and #line to guarantee callsite uniqueness